Phonebook.Frontend/src/app/pages/settings/settings.component.ts

```diff
@@ -1,41 +1,45 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
-import { ThemeService } from 'src/app/services/theme.service';
-import { LanguageService } from 'src/app/services/language.service';
-import { Language } from 'src/app/shared/models/enumerables/Language';
-import { I18n } from '@ngx-translate/i18n-polyfill';
-import { NotImplementedService } from 'src/app/modules/not-implemented/not-implemented.service';
-import { FeatureFlagService } from 'src/app/modules/feature-flag/feature-flag.service';
-import { MatSlideToggleChange } from '@angular/material/slide-toggle';
-import { Observable } from 'rxjs';
+import { Component, OnDestroy, OnInit } from "@angular/core";
+import { MatSlideToggleChange } from "@angular/material/slide-toggle";
+import { I18n } from "@ngx-translate/i18n-polyfill";
+import { Store, Select } from "@ngxs/store";
+import { Observable } from "rxjs";
+import { FeatureFlagService } from "src/app/modules/feature-flag/feature-flag.service";
+import { NotImplementedService } from "src/app/modules/not-implemented/not-implemented.service";
+import { LanguageService } from "src/app/services/language.service";
+import { Language } from "src/app/shared/models/enumerables/Language";
+import { AppState, SetTheme } from "src/app/shared/states";
+import { Theme } from "src/app/shared/models/enumerables/Theme";
 
 @Component({
-  selector: 'app-settings',
-  templateUrl: './settings.component.html',
-  styleUrls: ['./settings.component.scss']
+  selector: "app-settings",
```

Comment:
In die Projekt settings sollten wir doubleQuotes auf false setzen.

Reply:
Ich habe nichts gefunden wo man das richtig einstellen kann. In einer settings-Datei stand bereits, dass single quotes vorrangig verwendet werden sollen. Habe es jetzt noch zusätzlich bei mir in den prettier-Settings umgestellt